distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Pgpass.enso

```diff
@@ -21,7 +21,7 @@ polyglot java import java.lang.StringBuilder as Java_String_Builder
    - host: The hostname of the database server.
    - port: The port of the database server.
    - database: The database to connect to.
-read : Text -> Integer -> Text -> Text -> [Pair Text Text]
+read : Text -> Integer -> Text -> Text -> Vector Pair
```

Comment:
Let's keep the information from original signature, just with new syntax. Even if it's currently unchecked.

```suggestion
read : Text -> Integer -> Text -> Text -> Vector (Pair Text Text)
```